### ISSUE TITLE
`ocdev component set` should fail when the target component does not exist

### ISSUE THREAD

**Issue by containscafeine** [redhat-developer/ocdev#143]
```console
$ ocdev component set randomcomp
Switched to component: randomcomp
```

blocked on #53

**Comment by kadel** [comment]
this is no longer blocked

but it is duplicate of https://github.com/redhat-developer/ocdev/issues/127

**Comment by kadel** [comment]
Duplicate of #127
